### PR TITLE
Inquiry Detailed Page

### PR DESCRIPTION
**Pull request by dhaselhan**
* Detailed component
* Add Overview and Details Page

### CODE REVIEW

File: alcs-frontend/src/app/features/inquiry/inquiry.component.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+})
+export class InquiryComponent implements OnInit, OnDestroy {
+  $destroy = new Subject<void>();
+
+  inquiry?: InquiryDto;
+  fileNumber?: string;
+  childRoutes = childRoutes;
+
+  constructor(
+    private planningReviewService: InquiryDetailService,
```
Comment (mhuseinov) [inline alcs-frontend/src/app/features/inquiry/inquiry.component.ts:37]: the same for service name

File: alcs-frontend/src/app/features/inquiry/overview/overview.component.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Subject, takeUntil } from 'rxjs';
+import { InquiryDetailService } from '../../../services/inquiry/inquiry-detail.service';
+import { InquiryDto } from '../../../services/inquiry/inquiry.dto';
+import { TimelineEventDto } from '../../../services/planning-review/planning-review-timeline/planning-review-timeline.dto';
```
Comment (mhuseinov) [inline alcs-frontend/src/app/features/inquiry/overview/overview.component.ts:5]: is this a correct import?

**Review by mhuseinov — APPROVED** [review]
👍

File: alcs-frontend/src/app/features/inquiry/inquiry.component.ts [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  {
+    path: 'details',
+    menuTitle: 'Details',
+    icon: 'person',
+    component: DetailsComponent,
+  },
+];
+
+@Component({
+  selector: 'app-planning-review',
```
[the remark this answers is not in the document]
Comment (dhaselhan) [reply alcs-frontend/src/app/features/inquiry/inquiry.component.ts]: Fixed

File: alcs-frontend/src/app/features/inquiry/inquiry.component.ts (hunk 1 above)
Comment (dhaselhan) [reply alcs-frontend/src/app/features/inquiry/inquiry.component.ts]: Fixed

File: alcs-frontend/src/app/features/inquiry/overview/overview.component.ts (hunk 2 above)
Comment (dhaselhan) [reply alcs-frontend/src/app/features/inquiry/overview/overview.component.ts]: Removed the TimelineDto Import

**Pull request merged**